llvm/test/tools/llvm-objcopy/ELF/change-section-lma-single-load.test

```diff
@@ -0,0 +1,45 @@
+# RUN: yaml2obj %s -o %t
```

Comment:
It would be preferable to have all test cases in a single file. You can use yaml2obj's --docnum option to have multiple different YAML inputs in the same file (see other tests for examples).

That being said, what's the benefit of the "single-load" test case that the "multiple-load" case doesn't provide?

Reply:
One yaml file seems sufficient. The fixup has only one.